code/game/objects/items/skrell.dm

```diff
@@ -153,3 +153,57 @@
 
 		if(hologram_message)
 			visible_message("<span class='notice'>[hologram_message]</span>")
+
+/obj/item/jargontag
+	name = "\improper Jargon Federation loyalty ear-tag"
+	desc = "An ear-tag that shows the wearer is loyal to the Jargon Federation. A small cable travels into the ear canal..."
+	w_class = ITEMSIZE_SMALL
+	slot_flags = SLOT_EARS
+	icon = 'icons/obj/skrell_items.dmi'
+	icon_state = "jargtag"
+	item_state = "jargtag"
+	contained_sprite = TRUE
+	var/fried = FALSE // Doesn't work anymore
+
+/obj/item/jargontag/equipped(mob/living/carbon/human/M)
+	..()
+	if(fried)
+		return
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.l_ear == src || H.r_ear == src)
+			clamp_on(H)
+
+// Could add some stuff to this in the future? I dunno. I just couldn't figure out how to callback to_chat LOL - geeves
+/obj/item/jargontag/proc/do_loyalty(var/mob/wearer)
+	to_chat(wearer, span("good", "You feel an intense feeling of loyalty towards the Jargon Federation surge through your brain."))
+
+/obj/item/jargontag/proc/clamp_on(var/mob/wearer)
+	if(fried)
+		return
+	anchored = TRUE // good luck getting this bad boy off
```

Comment:
I don't know about this. If their head gets cut off then the tag will still be anchored once on the ground, meaning you can't move it.